lib/webtask.js

```diff
@@ -46,7 +46,14 @@ function Webtask (sandbox, token) {
     Object.defineProperty(this, 'url', {
         enumerable: true,
         get: function () {
-            var url = this.sandbox.url + '/api/run/' + this.sandbox.container;
+            var url;
+            if (this.claims.host) {
+               var surl = Url.parse(this.sandbox.url);
+               url = surl.protocol + '//' + this.claims.host + (surl.port ? (':' + surl.port) : '') + '/' + this.sandbox.container;
```

Comment:
So for development sandbox deployments, the port would need to be used on the custom host as well?